pyscriptjs/tests/integration/test_stdio_handling.py

```diff
@@ -146,6 +149,7 @@ async def coro(value, delay):
 
         self.assert_no_banners()
 
+    @pytest.mark.xfail(reason="fails after introducing synclink, fix me soon!")
```

Comment:
writing this comment as a reminder: we decided to fix these tests in a follow up PR

Reply:
follow up issue: https://github.com/pyscript/pyscript/issues/1314